test/medium-client-integration-tests.js

```diff
@@ -0,0 +1,120 @@
+var config = require("./load-config")
+var medium = require('../')
+var should = require('should')
+var testHelper = require('./test-helper')
+
+if (config.accessToken) {
+
+  var validateString = function (str, opt_expected) {
+    str.should.be.String().and.not.be.empty()
+    if (opt_expected) {
+      str.should.equal(opt_expected)
+    }
+  }
+
+  var validateArray = function (arr, opt_expected) {
+    arr.should.be.Array().and.not.be.empty()
+    if (opt_expected) {
+      arr.should.deepEqual(opt_expected)
+    }
+  }
+
+  describe("MediumClient", function () {
```

Comment:
Hmm .. realize this was a comment on unit tests attached to the file for integration tests. Well, integration tests are now gone and unit tests are now very thorough. Tension resolved.